updateClickOnce/task.json

```diff
@@ -91,6 +91,14 @@
         "required": false,
         "helpMarkDown": "The minimum version of this application a user can run. This flag makes the named version of your application a required update, use %version% to use the current version as minimum. The argument must be a valid version string of the format N.N.N.N"
       },
+      { 
+        "name": "ReplaceDeploy",
+        "type": "bool",
+        "label": "Replace Deploy File Extension",
+        "defaultValue": false,
```

Comment:
Maybe it’s best to have this as default true to maintain comp ability to current version.

Reply:
Updated to default to true